planner-api/openapi.yaml

```diff
@@ -78,6 +78,69 @@ paths:
         "401":
           description: "User unauthorized"
 
+  "/api/balance/sub":
+    post:
+      summary: "Creates new single subBalance"
+      requestBody:
+        description: "SubBalance Data"
+        required: true
+        content:
+          "application/json":
+            schema:
+              "$ref": "#/components/schemas/SubBalanceInfo"
+      responses:
+        "201":
+          description: "Created user subBalance"
+          content:
+            "application/json":
+              schema:
+                "$ref": "#/components/schemas/SubBalanceData"
+        "401":
+          description: "User unauthorized"
+
+    delete:
+      summary: "Delete all subBalances"
+      responses:
+        "204":
+          description: "Deleted subBalance"
+        "401":
+          description: "User unauthorized"
+
+  "/api/balance/sub/{id}":
+    get:
+      summary: "Get single subBalance"
+      requestBody:
```

Comment:
Why is there a requestBody? It's a GET method.

Reply:
Sorry for that little bit to much copy/paste work